### PR TITLE
fix: Ignored tests with no failures is success

### LINKED ISSUE #153: WebStorm: Test Skipped treated as Test Failure
**Describe the bug**

When running a suite with any skipped tests, the plugin shows a failure motivation event. Even though all of the non-skipped tests passes.

**To Reproduce**
Run [this test](https://github.com/Unthrottled/demo-js-app/blob/1bb1e87e9ea9336f3b97b65162006bb54b357697/src/App.test.js#L5)

**Expected behavior**
To show a success event, given no failures.

**Screenshots**
image

**Platform (please complete the following information):**
```
WebStorm 2020.2
Build #WS-202.6397.88, built on July 24, 2020
Licensed to Alex Simons
Subscription is active until February 21, 2021
Runtime version: 11.0.7+10-b944.20 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Windows 10 10.0
GC: ParNew, ConcurrentMarkSweep
Memory: 1979M
Cores: 12
Registry: ide.intellij.laf.enable.animation=true, ide.balloon.shadow.size=0
Non-Bundled Plugins: IdeaVIM, io.acari.DDLCTheme, io.acari.normandy.progress.bar, zd.zero.waifu-motivator-plugin
```
Plugin version: v1.2

### PR DESCRIPTION
**Pull request by Unthrottled**
# Motivation

Fixes #153 

## Notes

This wasn't just a WebStorm/Jest issue, if a any JUnit test was annotated with `@Ignore` would also cause this bug.

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -4,3 +4,8 @@ untilBuildVersion=202.*
 #
 pluginGroup=zd-zero
 pluginVersion=1.2
+
+# This property allows you
+# to run/develop the plugin on a different IDE
+# ex: idePath=/home/alex/.local/share/JetBrains/Toolbox/apps/WebStorm/ch-0/202.6250.10
```
Comment (zd-zero) [inline gradle.properties:10]: minor: Don't want to be nitpicky but this hits my OCD lol
"ex" is not really an abbreviation, request to change it to "e.g."

File: src/main/java/zd/zero/waifu/motivator/plugin/listeners/WaifuUnitTesterImpl.java [hunk 2]
```diff
@@ -41,6 +42,10 @@ public void stop() {
         this.busConnection.disconnect();
     }
 
+    private boolean isSuccessWithIgnoredTests( SMTestProxy.SMRootTestProxy testsRoot ) {
+        return TestStateInfo.Magnitude.IGNORED_INDEX.equals(  testsRoot.getMagnitudeInfo() );
+    }
+
```
Comment (zd-zero) [inline src/main/java/zd/zero/waifu/motivator/plugin/listeners/WaifuUnitTesterImpl.java:48]: Noice! A very simple way to resolve the issue 👌

File: gradle.properties (hunk 1 above)
Comment (Unthrottled) [reply gradle.properties]: in my head it was short for `example` 😄 

Small fixes

File: src/main/java/zd/zero/waifu/motivator/plugin/listeners/WaifuUnitTesterImpl.java (hunk 2 above)
Comment (Unthrottled) [reply src/main/java/zd/zero/waifu/motivator/plugin/listeners/WaifuUnitTesterImpl.java:48]: Yeah! I was happy that the fix was straightforward and involved no hacking :)

**Review by zd-zero — APPROVED** [review]

**Pull request merged**